srcpkgs/mosh/patches/ignore-unknown-renditions.patch

```diff
@@ -41,7 +41,7 @@ index a724afa6..93a31553 100644
    case 5: case 25: set_attribute(blink, value); break;
    case 7: case 27: set_attribute(inverse, value); break;
    case 8: case 28: set_attribute(invisible, value); break;
-+  default: assert(false);
++default: break;
```

Comment:
This is equal to not patching.

Reply:
But that patch is wrong, why would you crash mosh in this case?!

Reply:
Also it was reverted upstream in https://github.com/mobile-shell/mosh/commit/a40b67691aecfe9b31c3e8babf8e94d8d1a187a1

Reply:
It's funny, looking at the patch it shows break, but one applied with same title inserts assert(false)

Reply:
I removed the patch, and mosh+fish works